units/systemd-boot-check-no-failures.service.in

```diff
@@ -0,0 +1,24 @@
+#  SPDX-License-Identifier: LGPL-2.1+
+#
+#  This file is part of systemd.
+#
+#  systemd is free software; you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as published by
+#  the Free Software Foundation; either version 2.1 of the License, or
+#  (at your option) any later version.
+
+[Unit]
+Description=Check if Any System Units Failed
+Documentation=man:systemd-boot-check-no-failures.service(8)
+After=default.target graphical.target multi-user.target
```

Comment:
I suggest we accept @jwrdegoede's idea and add a `systemd-boot-complete-pre.target`, and order all checkers after it. Then we can add to the actual systemd.unit that is booted an implicit dependency `Before=systemd-boot-complete-pre.target`. Trying to list all possible targets here cannot work and has the disadvantage that if the admin boot to a different systemd.unit, this service can run early and return bogus results.

That new target could be documented as:
- it is ordered after the main transaction is finished
- checker units can order themselves to be after the main startup phase, but are also free to ignore it, if the state that they are checking is available earlier.